conf/messages.cy

```diff
@@ -2234,7 +2234,29 @@ manageBusinesses.type-of-property.uk                                     = Busne
 manageBusinesses.type-of-property.foreign                                = Busnes eiddo tramor
 manageBusinesses.type-of-property.error                                  = Dewiswch opsiwn
 
-# manage income sources reporting method
+# manage income sources reporting method (new - content FS R17 enabled)
+manageBusinesses.manage.propertyReportingMethod.new.heading.annual                  = Optio allan o’r cynllun Troi Treth yn Ddigidol ar gyfer Treth Incwm ar gyfer y flwyddyn dreth {0} i {1}
+manageBusinesses.manage.propertyReportingMethod.new.heading.quarterly               = Cofrestru ar gyfer y cynllun Troi Treth yn Ddigidol ar gyfer Treth Incwm ar gyfer y flwyddyn dreth {0} i {1}
+manageBusinesses.manage.propertyReportingMethod.new.description.annual              = Bydd hyn yn golygu nad oes angen i chi gyflwyno diweddariadau chwarterol mwyach trwy feddalwedd sy’n cydweddu ar gyfer y ffynhonnell incwm hon.
+manageBusinesses.manage.propertyReportingMethod.new.inset.annual                    = Os ydych eisoes wedi cyflwyno i CThEF unrhyw ddiweddariadau chwarterol ar gyfer y busnes newydd hwn ar gyfer y flwyddyn dreth hon, bydd angen i chi ailgyflwyno’r wybodaeth hon yn eich Ffurflen Dreth.
```

Comment:
This inset is wrong, it should be according to the ticket

"Os ydych eisoes wedi cyflwyno i CThEF unrhyw
ddiweddariadau chwarterol ar gyfer y busnes newydd
hwn ar gyfer y flwyddyn dreth hon, bydd angen i chi
ailgyflwyno’r wybodaeth hon yn eich Ffurflen Dreth."